slang-kotlin/src/test/java/com/sonarsource/slang/kotlin/KotlinParserTest.java

```diff
@@ -106,6 +108,29 @@ public void testMatchExpressions() {
     // FIXME check more complex cases when(x) { in 1..10 -> ; 1,2 ->
   }
 
+  @Test
```

Comment:
Can we add a test that asserts the trees of both slang and kotlin with comments similarly to testBinaryExpression? Note that we don't have support for Shebang comments in SLang syntax.